exporter/prometheus/prometheus_test.go

```diff
@@ -212,9 +129,9 @@ func TestCumulativenessFromHistograms(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to create prometheus exporter: %v", err)
 	}
-	view.RegisterExporter(exporter)
-	reportPeriod := time.Millisecond
-	view.SetReportingPeriod(reportPeriod)
+	//view.RegisterExporter(exporter)
```

Comment:
There are a few more commented lines that can be removed.

Reply:
removed all of them.